roles/docker/tasks/main.yml

```diff
@@ -67,7 +67,7 @@
     dest: /etc/sysconfig/docker
     regexp: '^{{ item.reg_conf_var }}=.*$'
     line: "{{ item.reg_conf_var }}='{{ item.reg_fact_val | oo_prepend_strings_in_list(item.reg_flag ~ ' ') | join(' ') }}'"
-  when: item.reg_fact_val != '' and  docker_check.stat.isreg
+  when: item.reg_fact_val != '' and  (docker_check.stat.isreg is define and docker_check.stat.isreg)
```

Comment:
The default filter should work in this case this `docker_check.stat` will always be present.

That would make it something like:
```
when: item.reg_fact_val != '' and  (docker_check.stat.isreg | default(false))
```